recipes-bsp/u-boot/README.md

```diff
@@ -59,6 +59,15 @@ apt-get install flashrom
 
 Attach the grabber to the flash chip, then run the following on the host machine:
 
+While the the size of the uboot file doesn't match the flash chip's size (16777216 B)
```

Comment:
"As the size ..." - or "if", but I think we don't ship or produce 16M images anywhere.

Reply:
Yes, i got it. I will use "if" instead of "while". Thank you very much.